requirements-app.txt

```diff
@@ -13,6 +13,8 @@ jsonschema==2.5.1
 lorem==0.1.1
 mailchimp3==2.0.11
 pandas==0.19.1
+pygithub==1.38
 python-dateutil==2.4.2
 unicodecsv==0.14.1
 xeger==0.3.1
+yq==2.4.1
```

Comment:
Can't see the direct dep on yq either?

Reply:
Gone. I think that got slurped in from my other PR for rotating creds.